integration/tests/cook/test_basic.py

```diff
@@ -25,8 +25,8 @@ def setUp(self):
         util.wait_for_cook(self.cook_url)
 
     def test_basic_submit(self):
-        job_uuid, resp = util.submit_job(self.cook_url)
-        self.assertEqual(resp.status_code, 201)
+        job_uuid, resp = util.submit_job(self.cook_url, executor='cook')
```

Comment:
I'm confused about why we're specifying `executor='cook'` here but then below we check for `util.is_using_cook_executor(self.cook_url)`